### PR TITLE
add jels60v2 support

### PR DESCRIPTION
**Pull request by Jels02**
## Description

add support for v2 of the PCB using rp2040

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jels/jels60/v2/info.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "pin": "GP29",
+        "driver": "vendor"
+    },
+    "rgblight": {
+        "led_count": 22,
+        "sleep": true,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "max_brightness": 255,
```
Comment (zvecr) [inline keyboards/jels/jels60/v2/info.json:29]:
```suggestion
```

File: keyboards/jels/jels60/v2/info.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "indicators": {
+        "caps_lock": "GP28"
+    },
+    "debounce": 5
```
Comment (zvecr) [inline keyboards/jels/jels60/v2/info.json:46]:
```suggestion
    }
```

File: keyboards/jels/jels60/v2/info.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "diode_direction": "COL2ROW",
```
Comment (zvecr) [inline keyboards/jels/jels60/v2/info.json:4]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]
> keyboard updates and refactors (eg. to data driven) must go through develop to reduce master -> develop merge conflicts

This would include adding to `data/mappings/keyboard_aliases.hjson` for `jels/jels60` to `jels/jels60/v1` compatibility.

**Comment by Jels02** [comment]
@zvecr would I be able to add a temporary keyboard (i.e jels/jels60v2) to master (so that I can change VIA) and then put in a pr for refactoring targeted towards develop?

**Comment by drashna** [comment]
> @zvecr would I be able to add a temporary keyboard (i.e jels/jels60v2) to master (so that I can change VIA) and then put in a pr for refactoring targeted towards develop?

IIRC, via doesn't actually provide firmware images.  The collection that they have is very much out of date, I believe.  So, really, it's up to you to provide them. So compiling from source means that you should be able to compile from develop. So shouldn't be an issue.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**